### ISSUE TITLE
Extension issue: Intellisense is not working for docker compose file.

### ISSUE THREAD

**Issue by ashish-thakur111** [microsoft/vscode-docker#3222]
- Issue Type: `Bug`
- Extension Name: `vscode-docker`
- Extension Version: `1.17.0`
- OS Version: `Windows_NT x64 10.0.19043`
- VS Code version: `1.60.1`

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 74,
		"activateCallTime": 3,
		"activateResolvedTime": 44,
		"activationReason": {
			"startup": false,
			"extensionId": {
				"value": "ms-azuretools.vscode-docker",
				"_lower": "ms-azuretools.vscode-docker"
			},
			"activationEvent": "onLanguage:dockercompose"
		}
	},
	"runtimeErrors": [],
	"runningLocation": 1
}
```
Intellisense is not showing up while writing docker compose file, even pressing ctrl+space key is not showing any suggestion.
Thanks

**Comment by bwateratmsft** [comment]
Hey @ashish-thakur111, this is by-design. We're in the process of implementing a proper language server for Compose. You can try it out by setting `docker.enableDockerComposeLanguageService` to `true`.

It's not very complete though, the completions right now are limited to just help with volumes. We're working on getting more for the next release.

**Comment by ashish-thakur111** [comment]
Thanks for info @bwateratmsft . I checked out the readme at https://github.com/microsoft/vscode-docker#docker-compose, had a impression that it is working, but not for me in my vscode.

**Comment by bwateratmsft** [comment]
🤦‍♂️

Somehow we forgot to update the readme! I feel silly...

**Comment by zak905** [comment]
Even if I set `docker.enableDockerComposeLanguageService` to `true` it did not work for me. I had to downgrad to version v1.15.0 (random choice, it could be working with next releases inferior to v1.17.0) to get it to work. VS Code version: 1.60.2

**Comment by clouedoc** [comment]
For other users looking to use rollback to a previous version:

<img width="541" alt="Capture d’écran 2021-10-13 à 21 20 45" src="https://user-images.githubusercontent.com/13921610/137199238-a7465e1b-04f7-4a1a-a60d-0cec987b2d92.png">

**Comment by bwateratmsft** [comment]
This fix is now released in Docker extension version 1.18.0. The Compose language service has been enabled by default and most of the completions present in 1.16 have returned. Some are not there (especially older Compose spec v2 ones), but we think the most important ones are present.
